torchvision/transforms/v2/_meta.py

```diff
@@ -55,21 +58,15 @@ class ConvertDtype(Transform):
 
     _v1_transform_cls = _transforms.ConvertImageDtype
 
-    _transformed_types = (is_simple_tensor, datapoints.Image, datapoints.Video)
+    _transformed_types = (is_simple_tensor, datapoints.Image)
 
     def __init__(self, dtype: torch.dtype = torch.float32) -> None:
         super().__init__()
         self.dtype = dtype
 
-    def _transform(
-        self, inpt: Union[datapoints._TensorImageType, datapoints._TensorVideoType], params: Dict[str, Any]
-    ) -> Union[datapoints._TensorImageType, datapoints._TensorVideoType]:
-        return F.convert_dtype(inpt, self.dtype)
-
-
-# We changed the name to align it with the new naming scheme. Still, `ConvertImageDtype` is
-# prevalent and well understood. Thus, we just alias it without deprecating the old name.
-ConvertImageDtype = ConvertDtype
+    def _transform(self, inpt: datapoints._TensorImageType, params: Dict[str, Any]) -> datapoints._TensorImageType:
+        output = F.to_dtype_image_tensor(inpt.as_subclass(torch.Tensor), self.dtype, scale=True)
+        return datapoints.Image.wrap_like(inpt, output)
```

Comment:
As written, this will convert plain tensor inputs into images. Should we just call the dispatcher and let it handle the (un-)wrapping?

```suggestion
        return F.to_dtype(inpt, dtype=self.dtype, scale=True)
```

Since we also set `_transformed_types` above, we'll only ever see plain tensors and images as `inpt`.